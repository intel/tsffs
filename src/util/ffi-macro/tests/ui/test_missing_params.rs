// Copyright (C) 2023 Intel Corporation
// SPDX-License-Identifier: Apache-2.0

use ffi_macro::callback_wrappers;

struct Test {}

#[callback_wrappers]
impl Test {
    pub fn test(&self, a: i32, b: i32) -> i32 {
        a + b
    }
}

fn main() {}
