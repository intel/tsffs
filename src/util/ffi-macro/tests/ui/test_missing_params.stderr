error: Expected `params` attribute
  --> tests/ui/test_missing_params.rs:10:5
   |
10 | /     pub fn test(&self, a: i32, b: i32) -> i32 {
11 | |         a + b
12 | |     }
   | |_____^
